src/data/sample/DivisionCountSampleDataset.ts

```diff
@@ -14,25 +14,37 @@ export class DivisionCountSampleData {
     };
   }
 
-  static countByDivisionGroup(data: DivisionCountSampleEntry[]): Map<string, number> {
+  static countByDivisionGroup(
+    data: DivisionCountSampleEntry[]
+  ): {
+    divisionData: Map<string, number>;
+    geoInfoArray: { country: string | null; region: string | null }[];
+  } {
+    let geoInfoArray: { country: string | null; region: string | null }[] = [];
     const output = new Map<string, number>();
     for (const entry of data) {
       if (entry.division === null) {
         continue;
```

Comment:
**Source of error!** This falsifies the country and region counts if a country/region does not associate every sequence with a division (which they don't do). If a country (for a given variant) does not provide any division information, that country will not show up in the table at all!